package.json

```diff
@@ -57,8 +57,8 @@
     "schemas": "echo 'The `schemas` script is deprecated. Please run `yarn codegen` instead.' && node scripts/codegen.js",
     "postinstall": "node -e 'process.exit(process.env.CI || process.env.SKIP_POSTINSTALL ? 0 : 1);' || node ./scripts/postinstall.js",
     "check-tags": "yarn ts scripts/check-tags.ts",
-    "charts": "cd ./infra && yarn charts",
-    "infra": "cd ./infra && yarn cli",
+    "charts": "cd ./infra && yarn install > /dev/null && yarn charts",
+    "infra": "cd ./infra && yarn install > /dev/null && yarn cli",
```

Comment:
Why add this install overhead when we are making charts?

Reply:
Hmm I ran yarn charts - I know you are hiding the output but actual packages weren't installed, after I ran charts, I ran yarn install and there were packages that were fetched for me.

Reply:
Because @robertaandersen pointed out that this failed and it is not obvious when doing this for the first time that you need to run yarn install in the infra folder. 

This extra step only happens locally.

Reply:
If yarn install has already run this only adds few ms overhead - it does not download or install everything again.